### DISCUSSION TITLE
点歌：Masked Autoencoders Are Scalable Vision Learners - Kaiming He

### DISCUSSION THREAD

**Discussion by green-dalii** [mli/paper-reading#45] [General]
> Abstract
This paper shows that masked autoencoders (MAE) are
scalable self-supervised learners for computer vision. Our
MAE approach is simple: we mask random patches of the
input image and reconstruct the missing pixels. It is based
on two core designs.

**Comment by green-dalii** [comment]
arxiv link:
[Masked Autoencoders Are Scalable Vision Learners](https://arxiv.org/pdf/2111.06377.pdf)

**Comment by mli** [comment]
会在后面讲，不过想把前面的相关工作讲一些先
